primitives/src/sentry.rs

```diff
@@ -22,9 +22,14 @@ pub use event::{Event, EventType, CLICK, IMPRESSION};
 
 #[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
 #[serde(rename_all = "camelCase")]
-/// Channel Accounting response.
+/// Channel Accounting response
+/// A collection of all `Accounting`s for a specific [`Channel`](`crate::Channel`)
```

Comment:
- This will ensure that the second line sentence is on a new line.
- The link in the `()` should not include backticks! It's just special markdown link
```suggestion
/// Channel Accounting response.
///
/// A collection of all `Accounting`s for a specific [`Channel`](crate::Channel).
```